Feature/inrel 6499 videoobject

killes opened the pull request.

killes committed:
Merge pull request #256 from BurdaMagazinOrg/8.x-3.x

Update feature branch

killes committed:
INREL-6499: add meta info to itemObject

killes committed:
INREL-6499: trim the created date

killes committed:
Revert "INREL-6499: trim the created date"

This reverts commit b2fe37bf2857e23cc8fcea01c1668b172258e728.

killes committed:
INREL-6499: only display description if set

IT-Cru commented:
Is the `if content.field_description|render` really needed? I think the description meta item should exists every time also when field_description is not filled with data. Perhaps spaceless than could be also removed or should come around ALL meta items.

If it is needed please replace with `if not media.field_desciption.isEmpty()` logic to avoid twice render processing.

killes committed:
INREL-6499: rm not needed condition, wrap everything in spaceless

killes commented:
> Is the `if content.field_description|render` really needed? I think the description meta item should exists every time also when field_description is not filled with data. Perhaps spaceless than could be also removed or should come around ALL meta items.

I've removed the condition and wrapped spaceless around everything.

IT-Cru commented:
LGTM
